ppafm/ocl/AFMulator.py

```diff
@@ -450,6 +531,64 @@ def plot_images(self, X, outdir='afm_images', prefix='df'):
         plotImages(os.path.join(outdir, prefix), X, slices = list(range(0, len(X))),
             zs=zTips, extent=extent, cmap=PPU.params['colorscale'])
 
+def _get_params(file_path):
+    '''Get AFMulator arguments from a params.ini file.'''
+    PPU.loadParams(file_path)
+    lvec = np.array([
+        PPU.params['FFgrid0'],
```

Comment:
loolin on `sample_lvec` and `lvec` side-by-side I'm thinking maybe it would be better to keep origin `FFgrid0` as the last not the first. It would allow to always read first 3 vectors as a,b,c latticle vectros, oblivious if there is 4th vector. 
....
But that is topic for an other issue, not for this PR.

Reply:
That could be useful, but also would affect quite a few place, so you would have to go carefully through all of the logic that uses `lvec`.